lib/rules/forbid-prop-types.js

```diff
@@ -170,6 +170,7 @@ module.exports = {
       CallExpression(node) {
         if (
           node.arguments.length > 0
+          && node.callee.object.name === 'PropTypes'
```

Comment:
this shouldn't check for `PropTypes` specifically - it should ensure that it was either imported/required from `prop-types`, or, that it came from `React.PropTypes`.

Reply:
Understood, will update.

Reply:
I think it should be `[pragma].PropTypes`.

Reply:
yes, thanks, good call.

Reply:
Thanks for some direction. The check now uses the `pragmaUtil` instead of checking against the name inside callee